### PR TITLE
Added grafana to playground cluster

### PR DESCRIPTION
**Pull request by Freedisch**
**Notes for Reviewers**

This PR fixes add Grafana to the playground cluster
mesh design: https://meshery.layer5.io/catalog/a459be4e-3945-4331-acf1-b9fec90846aa

PS: we need to verify if Prometheus and node exporter are still running in the playground cluster

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — COMMENTED** [review]
@Freedisch, will you:
1. confirm that you’re able to provision this infra in Playground (ns monitoring and grafana both present?)
2. confirm that you’re able to update Meshery Server’s Settings to connect to that running instance of Grafana.

File: install/playground/kubernetes/grafana.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: grafana
+  namespace: monitoring
+  annotations:
+    prometheus.io/scrape: 'true'
+    prometheus.io/port: '300'
```
Comment (leecalcote) [inline install/playground/kubernetes/grafana.yaml:66]: Prometheus will be exposed on 9090, right?

File: install/playground/readme.md [hunk 2]
```diff
@@ -67,6 +67,11 @@ Make sure to add nodeexporter.service inside /etc/systemd/system so that node ex
 After this create the namespace `monitoring` and apply `prometheus.yaml`.
 Notice the job_name: `prometheus` configured with two targets. It is the address of node exporter running on each node.
 
+## Grafana deployment
+Grafana uses prometheus as data source and expose it 3000. Prometheus and Node exporter should be deployed a long as with Grafana.
+
+Create the namespace `monitoring` if not already done and apply `grafana.yaml`.
+Notice that url in the data source should match with prometheus
```
Comment (leecalcote) [inline install/playground/readme.md:74]:
```suggestion
## Grafana Deployment
Grafana uses Prometheus as its data source. Grafana is exposed on port 3000/tcp. Prometheus and Node Exporter are prerequisites to Grafana's usefulness. 

Create the namespace `monitoring` if not already done and apply `grafana.yaml`.
Notice that URL in the data source should match with Prometheus.
```

File: install/playground/kubernetes/grafana.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  prometheus.yaml: | -
+    apiVersion: 1
+      datasources: [
+        {
+          "access": "proxy",
+          "editable": true,
+          "name": "prometheus",
+          "orgId": 1,
+          "type": "prometheus",
+          "url": "http://prometheus-service.monitoring.svc:8080",
```
Comment (leecalcote) [inline install/playground/kubernetes/grafana.yaml:16]:
```suggestion
          "url": "http://prometheus-service.monitoring.svc:9090",
```
9090?

**Comment by leecalcote** [comment]
@Freedisch what Grafana dashboards will be available in this Grafana deployment? We'll want a dashboard that is configured to collect the standard metrics available via Node Exporter.

**Comment by vishalvivekm** [comment]
@Freedisch Let's discuss this on Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Comment by Freedisch** [comment]
@leecalcote //

1- I’m able to provision the infra into the playground ✅
2- meshery's server cannot to Grafana, I believe this might be a network issue between my playground and Grafana 🚫 I'm still working on it
3- The charts that will provide Grafana are based on the metrics provided by Prometheus previously scraped from node exporter

Question: is the current deployment of Prometheus in the cluster accessible by Meshery's server? same question for node exporter?

**Comment by leecalcote** [comment]
@Freedisch I've spoken to this a few times and walked through the Meshery Architecture slide on the Playground's deployment networking. Please refer to that slide. Rephrase your questions in context of that slide, if you have any after studying it.

**Comment by Freedisch** [comment]
> @Freedisch I've spoken to this a few times and walked through the Meshery Architecture slide on the Playground's deployment networking. Please refer to that slide. Rephrase your questions in the context of that slide, if you have any after studying it.

I meant, is there any way we could verify if the Prometheus deployment still running inside the cluster? if not we should redeploy it.

**Comment by leecalcote** [comment]
When you open Visualizer mode, do you see it?

**Comment by Freedisch** [comment]
> When you open Visualizer mode, do you see it?

I Got it
I think we should have a documentation section for Prometheus setup into the playground

File: install/playground/kubernetes/grafana.yaml (hunk 1 above)
Comment (leecalcote) [reply install/playground/kubernetes/grafana.yaml:66]: @Freedisch ... following up.

Comment (Freedisch) [reply install/playground/kubernetes/grafana.yaml:66]: Yes, it will be exposed at 9090

**Pull request closed without merging**